aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -320,6 +320,13 @@ Tensor& remainder_(Tensor& self, const Tensor& other) {
 }
 
 Tensor& floor_divide_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  TORCH_WARN_ONCE(
```

Comment:
Take a look at https://github.com/pytorch/pytorch/pull/51745/files. This warning got a slight update and needs to be replicated for floor_divide(). You'll have to verify manually that the warning appears for the function, method, inplace, and out= variants.